fixbackend/auth/redirect_to_spa.py

```diff
@@ -28,32 +29,24 @@ class RedirectToSPA(Transport):
 
     scheme: SecurityBase  # not used
 
-    def __init__(self, redirect_path: str):
-        self.redirect_path = redirect_path
+    def __init__(self, redirect_url: str, ttl_seconds: int):
+        self.redirect_url = redirect_url
+        self.ttl_seconds = ttl_seconds
 
     async def get_login_response(self, token: str) -> Response:
-        payload = f"""
-    <!DOCTYPE html>
-    <html>
-        <head>
-            <script>
-                localStorage.setItem("fix-jwt", "{token}");
-                window.location.replace("{self.redirect_path}");
-            </script>
-        </head>
-        <body></body>
-    </html>"""
-
-        response = Response(content=payload, status_code=status.HTTP_200_OK, media_type="text/html")
-        response.set_cookie("fix.auth", value=token)
+        response = RedirectResponse(url=self.redirect_url, status_code=status.HTTP_303_SEE_OTHER)
```

Comment:
Why 303? Is this expected to handle post requests?

Reply:
Semantically 303 is more correct here. The requests coming here is GET from oauth callback URL.